py/mkrules.mk

```diff
@@ -53,7 +53,7 @@ $(BUILD)/%.o: %.s
 
 define compile_c
 $(ECHO) "CC $<"
-$(Q)$(CC) $(CFLAGS) -c -MD -o $@ $<
+$(Q)$(CC) $(CFLAGS) -c -MD -o $@ $< || (echo -e "See \033[1;31mhttps://github.com/micropython/micropython/wiki/Build-Troubleshooting\033[0m" && false)
```

Comment:
is it worth making the string a (make) variable in this file, so it doesn't need to be repeated?

Reply:
Done